### PR TITLE
Document the json-automate reporter

### PR DESCRIPTION
**Pull request by clintoncwolfe**
Provides basic documentation for the json-automate reporter.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by alexpop — APPROVED** [review]
Short and sweet, thank you Clinton!

File: docs-chef-io/content/inspec/reporters.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ Specifies the full path to the location of a JavaScript file that will be read a
 
 ## Automate Reporter
 
-The `automate` reporter type is a special reporter used with [Chef Automate](https://automate.chef.io/). To use this reporter you must pass in the correct configuration via a json config `--config`.
+The `automate` reporter type is a special reporter used with [Chef Automate](https://automate.chef.io/). It communicates over the network to send its results to Automate. To use this reporter you must pass in the correct configuration via a json config `--config`.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/reporters.md]:
```suggestion
The `automate` reporter type is a special reporter which will send its results over the network to [Chef Automate]({{< relref "/automate/">}}). To use this reporter you must pass in the correct configuration via a json config `--config`.
```

File: docs-chef-io/content/inspec/reporters.md [hunk 2]
```diff
@@ -218,3 +218,12 @@ environment where a `chef_guid` or `node_uuid` is already known to Chef InSpec.
 #### environment
 
 This will set the environment metadata for Automate.
+
+## Json-Automate Reporter
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/reporters.md]:
```suggestion
## JSON-Automate Reporter
```

File: docs-chef-io/content/inspec/reporters.md [hunk 3]
```diff
@@ -218,3 +218,12 @@ environment where a `chef_guid` or `node_uuid` is already known to Chef InSpec.
...
+
+The `json-automate` reporter is a special reporter that prepares the data format used by the Automate reporter. `json-automate` does not communicate on the network; rather it simply produces the JSON report format that Automate would be consuming. Notably, the report is based on the `json` reporter, with the following modifications:
+
+ * Controls that appear in child profiles are de-duplicated by ID, merging up into the parent profile.
+ * Child profiles are deleted, flattening the report.
+
+The `json-automate` reporter is primarily used for internal needs, but some users may find it useful if they wish for a JSON based reporter that merges controls.
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/reporters.md]:
```suggestion
The `json-automate` reporter is primarily used for internal needs, but some users may find it useful if they want a JSON based reporter that merges controls.
```

**Pull request merged**